validate/validate.go

```diff
@@ -490,6 +498,35 @@ func namespaceValid(ns rspec.Namespace) bool {
 	return true
 }
 
+func nestedValid(os, pathA, pathB string) bool {
+	if pathA == pathB {
+		return false
+	}
+
+	var purePathA, purePathB, sep string
+	if os == "windows" {
+		sep = "\\"
+	} else {
+		sep = "/"
+	}
+	purePathA = strings.TrimSuffix(pathA, sep)
+	purePathB = strings.TrimSuffix(pathB, sep)
```

Comment:
I [still][1] think these could be:

    pathA = strings.TrimSuffix(pathA, sep)

and I don't think you need the `sep == "/" && pathA == sep` qualifiers below.

[1]: https://github.com/opencontainers/runtime-tools/pull/256#discussion_r84948410